### ISSUE TITLE
[Admin] [Products] Unit value is changed when using coma separator for decimals

### ISSUE THREAD

**Issue by audez** [openfoodfoundation/openfoodnetwork#9474]
## Description
In the admin dashboard, when **importing** a product, the price will be changed if you use a coma separator for decimals in the units field. This doesn't seem to be the case when you create the product from the dashboard.

## Expected Behavior
The units value should remain the same than the one indicated in the imported CSV.

## Actual Behaviour
The final price is is rounded to the nearest value.
For example, if you type 2,3 it turns into 2.

## Steps to Reproduce
1. Connect to admin > Products tab > [Import tab](https://coopcircuits.fr/admin/product_import)
2. Click on "Browse"
3. Select a file with at least one of the product using a coma as decimal separator for the "units" column value
Ex. of test file : [test4.csv](https://github.com/openfoodfoundation/openfoodnetwork/files/9210506/test4.csv)
4. Just after the import, in the "Import validation overview", everything nice. Save.
5. In the product list, observe that your product unit value has changed

## Animated Gif/Screenshot
![Screen Shot 2022-07-28 at 16 15 34](https://user-images.githubusercontent.com/28386648/181540609-b5a1b767-50bc-4709-bd31-6824823c452b.png)
![Screen Shot 2022-07-28 at 16 15 47](https://user-images.githubusercontent.com/28386648/181540763-e1e4636a-8d07-4011-8193-8f3c0c38ccd3.png)
![Screen Shot 2022-07-28 at 16 15 57](https://user-images.githubusercontent.com/28386648/181540827-ad074f95-7c01-47ed-8bef-cbf1ca53dcfe.png)

## Workaround
You can use a dot instead of a coma.

## Severity
bug-s4: it's annoying, but you can use it

## Possible Fix
Enable the use of coma for the value unit field.

**Comment by RachL** [comment]
@audez this happens also when you convert to UTF-8 ? I'm surprised it's possible to distinguish between the coma used as separator and the coma of the number. I wonder if it's safer to not allow comas 🤷

**Comment by audez** [comment]
yep, in UTF-8. You mean a comma used as a separator in a string, and the comma used as separator in a number? I dont think there can be a confusion, as "price" and "units" are fields in which you only expect a number. IMO it's fine to allow commas, and it looks like it's already half handled: for units it's ok in both the "Import validation view" and in product creation. So I guess the thing would be to add the handling: 
- for units: from the "import validation view" to the product view
- for price: in import and product creation

but i might be wrong!
